plugins/newsdownloader.koplugin/main.lua

```diff
@@ -57,6 +57,15 @@ function NewsDownloader:addToMainMenu(menu_items)
                 callback = function() self:setCustomDownloadDirectory() end,
             },
             {
+                text = _("Settings"),
+                callback = function()
+                    UIManager:show(InfoMessage:new{
+                        text = T(_("To change feed (Atom/RSS) sources please manually edit configuration file:\n%1\n\nIt's very simple, contains comments and sample configuration."),
```

Comment:
in that case, "*the* configuration file"

The second sentence is grammatically broken. I propose to fix it by getting rid of the first part and/or splitting it up.

> It's very simple. It contains comments and a sample configuration to help you get started.

or simply

> It contains comments and a sample configuration to help you get started.

Alternatively you could fix up the sentence as is with something like `It's very simple and contains comments as well as sample configuration.` or `It's very simple, contains comments and provides sample configuration.` But I find that stylistically less attractive.